### PR TITLE
Make ingest a subcommand of cytominer-database.

### LINKED ISSUE #28: Make ingest a subcommand of cytominer-database
So that you can do 
$ cytominer-database ingest 

See http://click.pocoo.org/5/commands/

### PR DESCRIPTION
**Pull request by mcquin**
Resolves #28 

```
> cytominer-database help

Usage: cytominer-database [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  ingest  Import CSV files into a database.
```

```
> cytominer-database ingest --help

Usage: cytominer-database ingest [OPTIONS] SOURCE

  Import CSV files into a database.

  SOURCE is a directory containing subdirectories that contain CSV files.

Options:
  -c, --config-file PATH  Configuration file.
  --munge / --no-munge    True if the CSV files for individual compartments
                          have been merged into a single CSV file. If True,
                          the CSV will be split into one CSV per compartment.
  -o, --target PATH       Connection string for the database.
  --help                  Show this message and exit.
```

### CODE REVIEW

File: cytominer_database/commands/command_ingest.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+@click.option(
+    "--munge/--no-munge",
+    default=True,
+    help="""\
+True if the CSV files for individual compartments
+have been merged into a single CSV file. If True,
+the CSV will be split into one CSV per compartment.
+"""
+)
+@click.option(  # TODO: Is this an argument/required?
```
Comment (mcquin) [inline cytominer_database/commands/command_ingest.py]: @shntnu Is `--target` a required argument? It looks like there isn't a provided default. Should the command be:

```
cytominer-database [OPTIONS] SOURCE TARGET
```

Comment (shntnu) [reply cytominer_database/commands/command_ingest.py]: `--target` is required. The usage was specified incorrectly. It should have been

```
Usage: cytominer-database ingest [OPTIONS] SOURCE -o TARGET
```

**Review by shntnu — CHANGES_REQUESTED** [review]
Claire will update to make `target` a required argument

**Comment by mcquin** [comment]
@shntnu Updated! :)

**Review by shntnu — APPROVED** [review]

**Pull request merged**